src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -54,7 +61,7 @@ public BaseType getType() {
         }
 
         final String documentUrlValue = documentUrl.textValue();
-        final String urlPatternString = "http.*/documents/[A-Za-z0-9-]+(?:/binary)?";
+        final String urlPatternString = applicationParams.getValidDMDomain() + "/documents/[A-Za-z0-9-]+(?:/binary)?";
```

Comment:
Can we be sure this won't exclude valid domains?  Is there a way to check all existing domains in usage?

Reply:
I am only putting back the old code and unit test back in action. I think the existing unit tests should be sufficient to test the code. I have tested with my local docker setup to make sure the domain regex used works with exisiting domains in AAT and production.

Reply:
I suggest asking Mutlu for a review. He was working in this area: https://tools.hmcts.net/jira/browse/RDM-4146 https://tools.hmcts.net/jira/browse/RDM-4146 We don't want to introduce the bug again.

Reply:
@MSancaktutar can also review this PR if he need too, look a his comments in the JIRA ticket.

Reply:
Happy to bring back the validation.